sections/TradeScroller/TradeScroller.component.js

```diff
@@ -1,87 +1,48 @@
 import styles from './TradeScroller.module.scss'
 import Marquee from 'react-fast-marquee'
 import useSWR from 'swr'
+import React from 'react'
 
-function Profile() {
-  const { data, error } = useSWR('/api/user', fetcher)
-
-  if (error) return <div>failed to load</div>
-  if (!data) return <div>loading...</div>
-  return <div>hello {data.name}!</div>
-}
+export default function TradeScroller() {
+  const tweetFetcher = (url) => fetch(url).then((r) => r.json())
 
-function fetchTwitterFeed() {
-  // call api endpoint with fetch
-  // /api/twitter-feed
-}
+  const { data, error } = useSWR('/api/twitter-feed', tweetFetcher)
 
-export default function TradeScroller() {
-  const { data, error } = useSWR('/api/user', fetchTwitterFeed)
+  function formatTweet(text, id) {
+    let stringArray = text.split(' ')
+    let profitIndex = stringArray.findIndex((string) => {
+      return string.includes('%')
+    })
 
-  // if (error) return
-  // if (!data) return <div>Loading twitter feed</div>
+    let profitStringIndex = profitIndex + 1
 
+    return (
+      <React.Fragment key={id}>
+        <p>
+          {stringArray.map((string, index) => {
+            if (index === profitIndex || index === profitStringIndex) {
+              return <span className='text-green'>{string + ' '}</span>
+            } else {
+              return string + ' '
+            }
+          })}
+        </p>
+        <span className={styles.tweetDivider}>|</span>
+      </React.Fragment>
+    )
+  }
+  if (error) return
+  if (!data || data?.tweets?.length === 0)
```

Comment:
@cdwhitman I added some failsafes here because if the build pipeline or a dev does not have the env variables setup it crashes the front end. Instead it will just not show it. 

added data?.tweets?.length so that if data is undefined calling tweets.length will not throw a "cannot read length of undefined error"